worlds/generic/docs/plando_en.md

```diff
@@ -161,8 +161,44 @@ into any locations within the game slots named BobsSlaytheSpire and BobsRogueLeg
 
 ## Boss Plando
 
-As this is currently only supported by A Link to the Past, instead of finding an explanation here, please refer to the
-relevant guide: [A Link to the Past Plando Guide](/tutorial/A%20Link%20to%20the%20Past/plando/en)
+This is currently only supported by A Link to the Past and Kirby's Dream Land 3. Boss plando allows a player to place a 
+given boss within an arena. More specific information for boss plando in A Link to the Past can be found in 
+its [plando guide](/tutorial/A%20Link%20to%20the%20Past/plando/en).
+
+Boss plando takes in a list of instructions for placing bosses, separated by a semicolon `;`.
+There are three types of placement: direct, full, and shuffle.
+* Direct placement takes both an arena and a boss, and places the boss into that arena.
+  * `Eastern Palace-Trinexx`
+* Full placement will take a boss, and place it into as many remaining arenas as possible.
+  * `King Dedede`
+* Shuffle will fill any remaining arenas using a given boss shuffle option, typically to be used as the last instruction.
+  * `full`
+
+### Examples
+
+```yaml
+A Link to the Past:
```

Comment:
It now recreates the LttP docs here, rather than referring to its guide.

Reply:
I can definitely trim it some, but I think there should be some LttP examples within the main guide since that will still likely be the most prominent use of boss plando. Maybe just leave off the last LttP example?

Reply:
Well, I suppose this will  be stuck until someone who reviews docs looks at it.

Reply:
didn't mention it in my original review but i did look through this and it seems fine to me. the generic doc should definitely have examples in it and these are reasonable. yes the lttp doc also have boss shuffle examples but these are the only two games that support boss plando so it's still relevant imo.